wpeverest/everest-forms
# Fix/gutenberg evf block credit card display issue

### All Submissions:






### Changes proposed in this Pull Request:



Closes # .

### How to test the changes in this Pull Request:

1. Create a payment form 
2. Open gutenberg block editor
3. Select EVF plugin from the block area and select payment form that you created and you will see that credit card field successfully appears .
4. Successfully works

### Types of changes:




### Other information:




### Changelog entry
Fix - Gutenberg EVF block credit card display issue
> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
